build/args/ffmpeg.gn

```diff
@@ -4,3 +4,4 @@ is_component_ffmpeg = true
 is_official_build = true
 proprietary_codecs = false
 ffmpeg_branding = "Chromium"
+enable_dsyms = false
```

Comment:
hm, ideally we'd still generate symbols for this build, though i'm not sure we currently have a place to publish them.

seems like a good fix for now though 👍

... how come this didn't get caught by CI?